d2bs/kolbot/libs/bots/TeamDiablo.js

```diff
@@ -0,0 +1,462 @@
+/**
+ *    @filename    Teamdiablo.js
+ *    @author      Jaenster
+ *    @desc        Make a true party of a baalrun.
+ */
+
+/*
+ */
+function TeamDiablo() {
+    var diablo = {
```

Comment:
What is the purpose of the var diablo here? It seems to encapsulate the entire function, which "TeamDiablo" already does.